lms/static/sass/views/_teams.scss

```diff
@@ -0,0 +1,259 @@
+.view-teams {
+
+  // resets for existing styles
+  div.course-wrapper section.course-content {
+    padding: 0;
+
+    p {
+      margin-bottom: ($baseline/4);
+    }
+  }
+
+  .icon {
+    font-family: "FontAwesome";
+  }
+
+// layout
+  .page-header {
+    @include clearfix();
+    padding: $baseline $baseline ($baseline/2) $baseline;
+    border-bottom: 3px solid $m-blue-l1;
+  }
+
+  .header-main {
+    display: block;
+  }
+
+  .page-header.has-secondary {
+
+      .header-main {
+        display: inline-block;
+        width: flex-grid(8,12);
+      }
+
+      .header-secondary {
+        display: inline-block;
+        width: flex-grid(4,12);
+        @include text-align(right);
+      }
+  }
+
+// ui bits
+  .breadcrumbs {
+    @extend %t-copy-sub1;
+
+    .nav-item {
+      @include margin-right($baseline/2);
+      display: inline-block;
+    }
+
+    .icon {
+      @include margin-right($baseline/4);
+      display: inline-block;
+      color: $gray-l2;
+
+      @include rtl {
+          @include transform(rotateY(180deg));
+        }
+    }
+  }
+
+  .page-title {
+    @extend %t-title4;
+    margin-bottom: ($baseline/4);
+    text-transform: none;
+    color: $black;
+  }
+
+  .page-description {
+    @extend %t-copy-sub1;
+    margin-bottom: ($baseline/4);
+    color: $gray;
+  }
+
+  .header-search {
+
+    .search-label {
+        @extend %text-sr;
+    }
+
+    .search-field {
+        transition: all .25s ease-in-out;
+        border: 0;
+        border-bottom: 2px solid transparent;
+        padding: ($baseline/4) ($baseline/2);
+        font-family: inherit;
```

Comment:
all inputs have font-family: sans-serif! Nice. Real nice. So yeah, i put inherit so we get whatever is around it, in this case open sans.